src/router.ts

```diff
@@ -238,7 +241,8 @@ v2Router.put("/:name+/manifests/:reference", async (req, env: Env) => {
     throw new ServerError("Content type not defined", 400);
   }
 
-  const { name, reference } = req.params;
+  const { reference } = req.params;
+  let  name  = req.params.name.includes('/') ? req.params.name : `library/${req.params.name}`;
```

Comment:
nit: run these through prettier and use `const`!

Reply:
i have run prettier, also added it to the devDepedencies as it was missing